frontend/public/locales/en/translation.json

```diff
@@ -438,6 +440,7 @@
   "Allow host ports": "Allow host ports",
   "Allow privilege escalation": "Allow privilege escalation",
   "Allow privileged container": "Allow privileged container",
+  "Allowed Cluster service versions": "Allowed Cluster service versions",
```

Comment:
Is it a requirement to have "Cluster" start with a capital C? 
```suggestion
  "Allowed cluster service versions": "Allowed cluster service versions",
```

Reply:
I think so. Just left a comment

Reply:
Because the kind is `ClusterServiceVersion`, I thought capital letters would help the user connect this to something else they might be familiar with. But I think it should either be title case "Cluster Service Version" or all lowercase.

Reply:
Now it is `Allowed Cluster Service Versions`